Dockerfile.dev

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:xenial
+
+# gcc for cgo
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		g++ \
+		gcc \
+		libc6-dev \
+		make \
+		wget \
+		pkg-config \
+		git \
+		curl \
+		ca-certificates \
+		libseccomp-dev \
+		parallel \
+		uidmap \
```

Comment:
Please install the latest revision from source

https://github.com/genuinetools/img/blob/master/Dockerfile#L20-L24

Reply:
@AkihiroSuda will do

Reply:
Updated to build the newuidmap/newgidmap similar to what is happening for alpine.